monitor/monitor.go

```diff
@@ -130,6 +130,7 @@ func New(opts MonitorOptions, logger *log.Logger, clk clock.Clock) (*Monitor, er
 			stats:            sthStats,
 			client:           client,
 			logURI:           opts.LogURI,
+			stopChannel:      make(chan bool, 1),
```

Comment:
This is a super nit, but: Let's make this an unbuffered channel, since there's no reason to make it buffered. It just saves a little cognitive overhead for the next person to come along and say "why a buffer of 1?" Same below.

Reply:
Works for me! 2ffec64